src/features/projectsV2/ProjectListControls/microComponents/ViewModeTabs.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode } from 'react';
+import { useTranslations } from 'next-intl';
+import themeProperties from '../../../../theme/themeProperties';
+import styles from '../styles/ProjectListControls.module.scss';
+import ListIcon from '../../../../../public/assets/images/icons/projectV2/ListIcon';
+import { SetState } from '../../../common/types/common';
+import LocationIconOutline from '../../../../../public/assets/images/icons/projectV2/LocationIconOutline';
+import { ViewMode } from '../../../common/Layout/ProjectsLayout/MobileProjectsLayout';
+interface ViewModeTabsProps {
+  setIsFilterOpen: SetState<boolean> | undefined;
+  isSearching: boolean | undefined;
+  setSelectedMode: SetState<ViewMode> | undefined;
+  selectedMode: ViewMode | undefined;
+}
+
+interface TabItemProps {
+  selectedTab: ViewMode;
+  icon: ReactNode;
+  label: string | undefined;
+}
+const { light } = themeProperties;
+const getIconColor = (mode: ViewMode, selectMode: ViewMode) =>
+  mode === selectMode ? light.light : light.richBlack;
```

Comment:
Move the destructuring statement inside the function, as it is only needed there.